File: src/main/java/com/gmail/borlandlp/minigamesdtools/arena/commands/ArenaCommandWatcher.java
package com.gmail.borlandlp.minigamesdtools.arena.commands;

import com.gmail.borlandlp.minigamesdtools.arena.ArenaPhaseComponent;
import org.bukkit.event.player.PlayerCommandPreprocessEvent;

public interface ArenaCommandWatcher extends ArenaPhaseComponent {
    void onCommand(PlayerCommandPreprocessEvent event);
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/arena/commands/ArenaCommandWatcher.kt
package com.gmail.borlandlp.minigamesdtools.arena.commands

import com.gmail.borlandlp.minigamesdtools.arena.ArenaPhaseComponent
import org.bukkit.event.player.PlayerCommandPreprocessEvent

interface ArenaCommandWatcher : ArenaPhaseComponent {
    fun onCommand(event: PlayerCommandPreprocessEvent)
}